solidity/deploy/23_authorize_relay_maintainer.ts

```diff
@@ -0,0 +1,27 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  await execute(
+    "LightRelay",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "setAuthorizationStatus",
+    true
+  )
+
+  await execute(
+    "LightRelay",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "authorize",
+    deployer
+  )
+}
+
+export default func
+
+func.tags = ["AuthorizeLightRelayMaintainer"]
+func.dependencies = ["LightRelay"]
```

Comment:
Shouldn't it depend on `RetargetLightRelay`? We do not want to enable authorization before we do the first retarget.

Reply:
It should.
Fixed in e652e7e61f070fa573232761b15a87adddb5adc8.